core/dynamo.c

```diff
@@ -2763,21 +2765,24 @@ dr_app_start_helper(priv_mcontext_t *mc)
 }
 
 /* dummy routine that returns control to the app if it is currently
```

Comment:
style: s/dummy/Dummy/.  I guess ignore if you do not want to change anything.

Reply:
I figured I would leave the comment like this since that's how it was before this change but I'm happy to clean up.

Reply:
The style guide requires capitalization only for multi-sentence comments, so your change of adding another sentence is what makes the original need to be modified.